folktables/folktables.py

```diff
@@ -141,7 +141,42 @@ def df_to_pandas(self, df, categories=None, dummies=False):
 
         return variables, target, group
 
-    @property 
+    def generate_categories(self, definition_df):
```

Comment:
If I understand correctly, this function is specific to ACS PUMS. In that case, it should not go into `folktables.py` and especially not as a method of `BasicProblem`. This file contains general code that is not specific to a single data source. Likewise the class `BasicProblem` needs to be data source agnostic.

I suggest we move this method into `load_acs.py`. Does this make sense?

Reply:
Thank you for the feedback. I understand that the current location of the `generate_categories` method is not ideal. My reasoning was that this code uses the `features` attribute of the specific problem and is thus problem specific.
I can move this functionality by requiring the user to submit the features themselves.
I would however suggest to move this into `acs.py` instead of `load_acs.py`. `load_acs.py` seems to have no exposed methods to a user and seems to mainly contain code downloading files from the Census website.

Are you fine with this approach?

Reply:
I have currently placed it in ```load_acs.py``` and exposed it to the user in the ```_init_.py``` file.